cabal.project

```diff
@@ -90,3 +87,20 @@ benchmarks: true
 
 -- The only sensible test display option
 test-show-details: streaming
+
+if impl(ghc < 9.8)
+  program-options
+    ghc-options: -Werror
```

Comment:
This is needed because partial functions like `head` and `tail` now emit a warning.

Will need to be removed when the code issues are fixed.